Error: Cannot find module 'package.json'

![image](https://user-images.githubusercontent.com/8336744/118075438-93e26280-b3e2-11eb-88f8-28502aae5a04.png)

It runs failed on GitLab runner.

![image](https://user-images.githubusercontent.com/8336744/118075501-a9578c80-b3e2-11eb-8947-4678676d78c8.png)


related https://github.com/shashkovdanil/clean-publish/issues/110

@antonk52 Sorry I was busy these days.

I've just tried your debug codes, and here is the logs:

```log
common info {
  cwd: '/builds/frontend/custom-webpack',
  version: 'v12.18.4',
  arch: 'x64',
  platform: 'linux'
}
search error: Error: Cannot find module 'package.json'
Require stack:
- /builds/frontend/custom-webpack/node_modules/lilconfig/dist/index.js
- /builds/frontend/custom-webpack/test.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.search (/builds/frontend/custom-webpack/node_modules/lilconfig/dist/index.js:108:33)
    at async main (/builds/frontend/custom-webpack/test.js:11:18) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/builds/frontend/custom-webpack/node_modules/lilconfig/dist/index.js',
    '/builds/frontend/custom-webpack/test.js'
  ]
}
{ result: undefined }
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -58,7 +58,14 @@ function getSearchPaths(startDir: string, stopDir: string): string[] {
         .reduceRight<{searchPlaces: string[]; passedStopDir: boolean}>(
             (acc, _, ind, arr) => {
                 const currentPath = arr.slice(0, ind + 1).join(path.sep);
-                if (!acc.passedStopDir) acc.searchPlaces.push(currentPath);
+                /**
+                 * fix #17
+                 * On *nix, if cwd is not under homedir,
+                 * the last path will be '', ('/build' -> '')
+                 * but it should be '/' actually.
+                 * And on Windows, this will never happen. ('C:\build' -> 'C:')
+                 */
+                if (!acc.passedStopDir) acc.searchPlaces.push(currentPath || path.sep);
                 if (currentPath === stopDir) acc.passedStopDir = true;
                 return acc;
             },

